application-mocca-calendar-api/src/main/java/org/xwiki/contrib/moccacalendar/internal/importJob/CalendarEventProcessor.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.moccacalendar.internal.importJob;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.contrib.moccacalendar.importJob.result.MoccaCalendarEventResult;
+
+import net.fortuna.ical4j.model.Property;
+import net.fortuna.ical4j.model.PropertyList;
+import net.fortuna.ical4j.model.component.CalendarComponent;
+
+/**
+ * Ical file import helper class for processing an event.
```

Comment:
I'll give more details here. If maybe you are not very familiar with some features of an application, this is supposed to give you an idea of what it does or how could it be used without looking around (as much as it can be done, of course; too much detail can also be bad). 
For example you can state here that it helps with processing the data from a CalendarComponent for mapping it to a MoccaCalendarEvent, mentioning also what a CalendarComponent is (similar to what you specified for `processEvent`) and how is it linked to the `ical file import`